dd-sdk-android-core/src/main/kotlin/com/datadog/android/core/internal/utils/RuntimeUtils.kt

```diff
@@ -7,52 +7,6 @@
 package com.datadog.android.core.internal.utils
 
 import com.datadog.android.v2.api.InternalLogger
-import java.util.Locale
 
 // Use it only when there is no way to access the SDK-specific logger.
 internal var unboundInternalLogger: InternalLogger = InternalLogger.UNBOUND
-
-/**
- * Warns the user that they're using a deprecated feature.
- * @param target the target feature (e.g. method name)
- * @param deprecatedSince the version when the feature was deprecated
- * @param removedInVersion the version in which the feature will disappear
- * @param alternative an alternative option to get the same effect
- */
-internal fun warnDeprecated(
```

Comment:
True, but then we might want to use a specific internal logger instead of the unbound one, so my thought was we can create a helper method then if need be.
One that can also warn us of usage of deprecated features to know when we can actually remove them